### PR TITLE
docs: add instruction how to disable gzip compression

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

add instruction how to disable gzip compression

---

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/docs/Troubleshoot_Collection.md [hunk 1]
```diff
@@ -389,3 +389,41 @@ Node configuration can be verified by following annotations:
...
+
+If you can see the following errors from Fluentd pods:
+
+```console
+2021-01-18 15:47:23 +0000 [warn]: #0 [sumologic.endpoint.logs.gc] failed to flush the buffer. retry_time=3 next_retry_seconds=2021-01-18 15:47:27 +0000 chunk="5b92e97a5ee3cbd7e59859644d9686e3" error_class=Zlib::GzipFile::Error error="not in gzip format"
+```
+
+Please see [migration topic on that](./v2_migration_doc.md#gzip-compression-errors).
+
+If the problem still occures, plesae disable gzip compression for buffer.
```
Comment (astencel-sumo) [inline deploy/docs/Troubleshoot_Collection.md:403]:
```suggestion
If the problem still occurs, please disable gzip compression for buffer.
```

**Review by astencel-sumo — APPROVED** [review]

File: deploy/docs/Troubleshoot_Collection.md [hunk 2]
```diff
@@ -389,3 +389,41 @@ Node configuration can be verified by following annotations:
 - `machineconfiguration.openshift.io/currentConfig`
 - `machineconfiguration.openshift.io/desiredConfig`
 - `machineconfiguration.openshift.io/state`
+
+### Gzip compression errors
+
+If you can see the following errors from Fluentd pods:
```
Comment (pmalek-sumo) [inline deploy/docs/Troubleshoot_Collection.md:395]:
```suggestion
If you observe the following errors from Fluentd pods:
```

**Comment by sumo-drosiek** [comment]
Related to #1522

File: deploy/docs/Troubleshoot_Collection.md [hunk 3]
```diff
@@ -389,3 +389,41 @@ Node configuration can be verified by following annotations:
...
+and the pod that emits the error is named `sumologic-fluentd-logs-1`,
+run the following set of commands:
+
+```bash
+NAMESPACE_NAME=collection
+
+for POD_NAME in $(kubectl get pods --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' | grep fluentd); do
+  kubectl -n ${NAMESPACE_NAME} delete pvc "buffer-${POD_NAME}" &
+  kubectl -n ${NAMESPACE_NAME} delete pod ${POD_NAME}
+  kubectl -n ${NAMESPACE_NAME} delete pod ${POD_NAME}
```
Comment (pmalek-sumo) [inline deploy/docs/Troubleshoot_Collection.md:424]: Why do we need to call this twice?

**Review by pmalek-sumo — CHANGES_REQUESTED** [review]

File: deploy/docs/Troubleshoot_Collection.md (hunk 3 above)
Comment (sumo-drosiek) [reply deploy/docs/Troubleshoot_Collection.md:424]:
```
The duplicated pod deletion command is there to make sure the pod is not stuck in `Pending` state
with event `persistentvolumeclaim "buffer-sumologic-fluentd-logs-1" not found`.
```

Comment (sumo-drosiek) [reply deploy/docs/Troubleshoot_Collection.md:424]: looks like race condition. I trust this has been wrote on purpose in first place, also we have explanation below

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**